modules/transactions/src/main/java/org/apache/ignite/internal/tx/TxMeta.java

```diff
@@ -24,13 +24,15 @@
 import org.apache.ignite.internal.hlc.HybridTimestamp;
 import org.apache.ignite.internal.replicator.ReplicationGroupId;
 import org.apache.ignite.internal.tostring.S;
+import org.jetbrains.annotations.Nullable;
 
 /** Transaction meta. */
 public class TxMeta implements Serializable {
     /** Serial version UID. */
     private static final long serialVersionUID = -172513482743911860L;
 
     /** Tx state. */
+    @Nullable
```

Comment:
What does `null` mean in this context? Is it really nullable?

Reply:
I picked up the idea about nullable from tests as in one place there was code like `new TxMeta(null, ...)`. It turned out to be wrong, removed the annotation.